### PR TITLE
Stabalize On Premise with External Minion Connection

### PR DESCRIPTION
**Pull request by jaymberry**
## Description

Stabalizing the on-premise solution, being able to add an external minion through the ingress, make install-local.sh easier to understand, and update Admin Guide (https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin).

## Jira link(s)
- https://opennms.atlassian.net/browse/HS-706

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.

### CODE REVIEW

**Comment by geraldhumphries** [comment]
Can you update this PR's description with the purpose? I assume `selector` is being removed because it's a Service resource concept, and not for Ingresses. Is that right?

**Review by geraldhumphries — CHANGES_REQUESTED** [review]

File: charts/opennms/templates/opennms/minion/minion-gateway-ingress.yaml [hunk 1]
```diff
@@ -23,5 +23,5 @@ spec:
                 port:
                   number: {{ .Values.OpenNMS.MinionGateway.GrpcPort }}
 
-  selector:
-    app: {{ .Values.OpenNMS.MinionGateway.ServiceName }}
+  #selector:
```
[the remark this answers is not in the document]
Comment (jaymberry) [reply charts/opennms/templates/opennms/minion/minion-gateway-ingress.yaml:26]: Yes, will do that right away.

**Comment by jaymberry** [comment]
> Can you update this PR's description with the purpose? I assume `selector` is being removed because it's a Service resource concept, and not for Ingresses. Is that right?

Yes, sorry, updated description and PR title.

**Review by geraldhumphries — APPROVED** [review]

**Review by splatch — APPROVED** [review]
I had initially some troubles with running it but after making proper clean start all worked as expected. One small suggestion is about marking `id.cfg` and `grpc.cfg` as examples on how to provision custom configurations into system, as these are 1:1 to what we currently include in minion image itself.

**Pull request merged**